packages/blade/src/components/Avatar/_decisions/decisions.md

```diff
@@ -0,0 +1,170 @@
+
+
+## Design
+
+- [Explore the design in Figma: Avatar](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=88229-1518352&mode=design&t=Gp3eolSGw8SybZkM-11)
+
+## API
+
+The `Avatar` & `AvatarGroup` components would have the following props:
+
+```ts
+type AvatarGroupProps = {
+  /**
+   * Children elements representing the avatars to stack.
+   */
+  children: React.ReactNode;
+  /**
+   * The size of each avatar within the group. Propagates to all avatars.
+   */
+  size?: 'xsmall' | 'small' | 'medium' | 'large';
+  /**
+   * The maximum number of avatars to display before truncating.
+   */
+  maxAvatars?: number;
+};
+
+type AvatarProps = {
+  /**
+   * The content or children of the avatar.
+   */
+  children?: StringChildrenType;
+  /**
+   * The size of the avatar.
+   */
+  size?: 'xsmall' | 'small' | 'medium' | 'large';
+  /**
+   * The visual variant of the avatar.
+   */
+  variant?: 'circle' | 'square';
+  /**
+   * The color theme of the avatar.
+   */
+  color?: 'primary' | 'positive' | 'negative' | 'notice' | 'information' | 'neutral';
+  /**
+   * Custom icon component to use as the avatar.
+   */
+  icon?: IconComponent;
+  /**
+   * The name of the avatar, used to generate initials.
+   * If src has loaded, the name will be used as the alt attribute of the img. If src is not loaded, the name will be used to create the initials.
+   */
+  name?: string;
+  /**
+   * Custom image source for an image avatar.
+   */
+  src?: string;
+  /**
+   * The `srcSet` attribute for the `img` element, useful for responsive images.
+   */
+  srcSet?: string;
+  /**
+   * The `alt` attribute for the `img` element
+   */
+  alt?: string;
+  /**
+   * CORS settings attributes
+   */
+  crossOrigin?: 'anonymous' | 'use-credentials' | '';
+  /**
+   * Automatically renders button with `a` tag with `href` on web
+   */
+  href?: ButtonProps['href'];
+  /**
+   * anchor target attribute
+   *
+   * Should only be used alongside `href`
+   */
+  target?: ButtonProps['target'];
+  /**
+   * anchor rel attribute
+   *
+   * Should only be used alongside `href`
+   */
+  rel?: ButtonProps['rel'];
+  /**
+   * Click handler for the avatar.
+   */
+  onClick?: Platform.Select<{
+    native: (event: GestureResponderEvent) => void;
+    web: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  }>;
+};
+```
+
+## Usage
+
+### Image Avatars
+
+```tsx
+<Avatar name="Remy Sharp" src="/static/images/avatar/1.jpg" />
+<Avatar name="Travis Howard" src="/static/images/avatar/2.jpg" />
+<Avatar name="Cindy Baker" src="/static/images/avatar/3.jpg" />
+```
+
+#### Avatar Fallbacks
+
+If there is an error loading the src of the avatar, there will be 2 fallbacks:
+
+- If there's a `name` prop, we use it to generate the initials.
+  - Avatar will always have 2 letters as initials.
+  - `Rama Krushna Behra` - `RK`
+  - `Nitin` - `NI`
+  - `Anurag Hazra` - `AH`
+- If there's no `name` prop, we use a default avatar.
```

Comment:
Should we make the default avatar configurable?

Reply:
Only the `color` is configurable, the  icon remains the same.

<img width="568" alt="Screenshot 2024-05-09 at 2 31 55 PM" src="https://github.com/razorpay/blade/assets/46647141/6382a20f-2624-4671-aaf1-c2ef94bcc2bd">